### PR TITLE
Adds an Elyran hijab to the code and loadout

### PR DESCRIPTION
**Pull request by feartheblackout**
Sprited by me after some hijinks in the Relay. Feel free to suggest changes.
hijabdisplay

### CODE REVIEW

**Comment by feartheblackout** [comment]
!review

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 1]
```diff
@@ -254,6 +254,7 @@
 	hijab["green hijab"] = /obj/item/clothing/head/hijab/green
 	hijab["blue hijab"] = /obj/item/clothing/head/hijab/blue
 	hijab["black hijab"] = /obj/item/clothing/head/hijab/black
+	hijab["elyran hijab"] = /obj/item/clothing/head/hijab/elyra
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_head.dm:257]:
```suggestion
	hijab["Elyran hijab"] = /obj/item/clothing/head/hijab/elyra
```

File: code/modules/clothing/head/misc.dm [hunk 2]
```diff
@@ -330,6 +330,12 @@
 	icon_state = "hijab_black"
 	item_state = "hijab_black"
 
+/obj/item/clothing/head/hijab/elyra
+	name = "elyran hijab"
```
Comment (SleepyGemmy) [inline code/modules/clothing/head/misc.dm:334]:
```suggestion
	name = "\improper Elyran hijab"
```

**Comment by atteria** [comment]
The color palette looks a bit deep-fried, to be honest.
Very saturated, very outline-heavy. There's noisy detailing on the inside that doesn't really read as anything, and is overpowered by said outline.
The elyran symbol on the back doesn't really read as it. I only realized it was there after checking discord.

**Pull request closed without merging**